README.md

```diff
@@ -49,6 +49,26 @@ COPY your-application.war ROOT.war
 The Tomcat instance can be configured through the environment variable `TOMCAT_PROPERTIES` which is
 a comma-separated list of `name=value` pairs appended to `catalina.properties`.
 
+### Stackdriver Logging
```

Comment:
Can you move it under "Optional Modules" and rename "Optional Modules" to "Optional Features"?